plugins/inputs/kapacitor/README.md

```diff
@@ -96,6 +104,36 @@ The total number of Kapacitor tasks.
 
 ---
 
+### kapacitor_alert
+The `kapacitor_alert` measurement stores fields with information related to
+[Kapacitor alerts](https://docs.influxdata.com/kapacitor/v1.5/working/alerts/).
+
+#### notification-dropped
+The number of alert or event handlers that failed.
```

Comment:
This is a bit misleading  as the `notification` in this context is not the alert handler but an internal notification used to deduplicate alerts. 

I would recommend something along these lines:

```
The number of internal notifications that were dropped because they arrived too late from another Kapacitor node. If this count is increasing it indicates that Kapacitor Enterprise nodes are not able to communicate fast enough compared to the volume of alerts.
```